TripJournal/settings.py

```diff
@@ -123,6 +123,12 @@
     'django.contrib.auth.backends.ModelBackend',
 )
 
+CONFIGS = {
```

Comment:
No point to name something CONFIGS in a file settings.py. Give it a name that corresponds to meaning. Like "AUTH_BY_EMAIL".